### PR TITLE
Added instructions to create dev namespace to Getting Started section.

### PR DESCRIPTION
**Pull request by borkod**
As I was going through the getting started section for #175 , I thought it would be helpful to have the step to create the `dev` namespace.

I also fixed toc as kubectl plugin is not related to installing from binary release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,11 @@
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```
Comment (danielpacak) [inline README.md:15]: I'm not an expert on that but is it useful? Most of the time I don't see the TOC being added to the TOC /cc @lizrice

**Review by danielpacak — CHANGES_REQUESTED** [review]
👋 @borkod I like the step that creates the namespace. For other changes please read my comments.

File: README.md [hunk 2]
```diff
@@ -56,7 +57,7 @@ binary versions can be manually downloaded and installed.
 
 From there, you should be able to run Starboard CLI commands: `starboard help`
 
-#### kubectl plugin
+### As a kubectl plugin
```
[the remark this answers is not in the document]
Comment (borkod) [reply README.md:60]: Sounds good. I'll revert the change.

File: README.md (hunk 1 above)
Comment (borkod) [reply README.md:15]: I didn't notice this. I have VS code extension for Markdown and it automatically generates TOC. I can revert this change too if you wish.

**Review by danielpacak — APPROVED** [review]

**Pull request merged**